includes/class-wcpdf-main.php

```diff
@@ -176,6 +176,13 @@ public function attach_pdf_to_email ( $attachments, $email_id, $order, $email =
 				$attachments[] = $pdf_path;
 
 				do_action( 'wpo_wcpdf_email_attachment', $pdf_path, $document_type, $document );
+
+				// log document creation to order notes
+				$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
```

Comment:
this is already loaded at init here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/830d26ae6a63d029c1763d6a12dbd95dd170beeb/includes/class-wcpdf-settings.php#L34
so you can avoid an extra db call